pkg/containerd/events.go

```diff
@@ -191,6 +217,30 @@ func (e *EventMonitor) createPortMapping(ctx context.Context, containerID string
 	return portMap, nil
 }
 
+func updateListener(ctx context.Context, portMappings nat.PortMap, action func(context.Context, net.IP, int) error) {
+	for _, portBindings := range portMappings {
+		for _, portBinding := range portBindings {
+			port, err := strconv.Atoi(portBinding.HostPort)
+			if err != nil {
+				log.Errorf("port conversion for [%s] error: %v", portBinding.HostPort, err)
```

Comment:
Instead of just printing `HostPort`, maybe print out the whole `portBinding` so it's easier to figure out what it's in reference to? (I'm thinking `%+v`.)